back command - rewinds n number of frames

echophon opened the pull request:
similar to time command, but rewinds an amount of frames from the current position to create stutters and other rhythmic effects

neauoire commented:
Can you call it `rewind`, and put it on `line 12`.
Also, don't use a method to get the frame, just do `.f`

echophon commented:
renamed, added a forward command 'skip'

example usage 
```
7U8..................
.*:01D...............
3u8..................
..:01C......gC.......
5ub..........44TCAFE.
..:01F.......aVC.....
.....................
.....................
.....C8..Cb..Cg......
.....4F0.9F0.4F0.....
.....................
......J...J...J...Va.
...............:13C..
......J...J...Va.....
...........:14C......
......J...Va.........
.......:14C..........
.....................
.....2D8.............
.......;rewind:0.....
.....................
......D8.............
.......;skip:0.......
```
